src/bot/commands/public/settings.command.ts

```diff
@@ -1,198 +1,43 @@
-    this.settingsMenuObj = new MiddlewareMenu('settingsMenu', { autoAnswer: false })
-      .addGlobalMiddlewares(onlyAdmin)
-      .text(deleteMessageButton, (context) => toggleSetting(context, 'disableDeleteMessage'))
-      .text(deleteTensorButton, (context) => toggleSetting(context, 'disableStrategicInfo'))
-      .text(deleteSwindlerButton, (context) => toggleSetting(context, 'disableSwindlerMessage'))
-      .row()
-      .text(deleteNsfwButton, (context) => toggleSetting(context, 'disableNsfwFilter'))
-      .row()
-      .text(deleteCardsButton, (context) => toggleSetting(context, 'enableDeleteCards'))
-      .text(deleteUrlsButton, (context) => toggleSetting(context, 'enableDeleteUrls'))
-      .text(deleteLocationsButton, (context) => toggleSetting(context, 'enableDeleteLocations'))
-      .row()
-      .text(deleteMentionButton, (context) => toggleSetting(context, 'enableDeleteMentions'))
-      .text(deleteForwardedButton, (context) => toggleSetting(context, 'enableDeleteForwards'))
-      .text(deleteServiceMessageButton, (context) => toggleSetting(context, 'disableDeleteServiceMessage'))
-      .row()
-      .text(warnRussianLanguageButton, (context) => toggleSetting(context, 'enableWarnRussian'))
-      .text(deleteRussianLanguageButton, (context) => toggleSetting(context, 'enableDeleteRussian'))
-      .text(deleteCounteroffensiveButton, (context) => toggleSetting(context, 'enableDeleteCounteroffensive'))
-      .row()
-      .text(airAlarmAlertButton, isAlarmNow, async (context) => {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
-        context.menu.nav('settingsAirRaidAlertSubmenu');
-        await context.editMessageText(getAirRaidAlarmSettingsMessage(context.chatSession.chatSettings), { parse_mode: 'HTML' });
-      })
-      .text(airAlarmNotificationMessage, isAlarmNow, isStateSelected, async (context) => {
-        context.chatSession.chatSettings.airRaidAlertSettings.notificationMessage =
-          !context.chatSession.chatSettings.airRaidAlertSettings.notificationMessage;
-        alarmChatService.updateChat(context.chatSession, context.chat?.id);
-        const newText = getSettingsMenuMessage(context.chatSession.chatSettings);
+      if (isChatPrivate) {
+        const userSession = await redisService.getUserSession(userId);
+        const chats = userSession?.linkedChats || [];
+        return chats.length > 0
+          ? context.reply(linkToWebView)
+          : context.reply(`You don't linked chats yet. Please, go in group and press /settings.`);
+      }
 
-        if (context.msg?.text !== newText) {
-          await context.editMessageText(newText, { parse_mode: 'HTML' });
+      if (!isChatPrivate) {
+        const admins = await context.api.getChatAdministrators(chatId);
+        if (!admins.some((object) => object.user.id.toString() === userId)) {
```

Comment:
1) It's hard to read, it's better to move into a variable called `isNotAdmin`;
2) Don't name array method arguments as `object` or any other abstract name. It's hard to understand when we have a defined name. I suggest you name it as `admin`